### DISCUSSION TITLE
more speed

### DISCUSSION THREAD

**Discussion by khosro10k** [danielgatis/rembg#405] [Q&A]
Hello . This application is great. Is there a way to make it work faster? I would be grateful if you introduce a faster version or provide a solution to make this application faster.

**Comment by anaalmer** [comment]
Hi! I'm looking for the same thing... do you work with CUDA or CPU?

**Reply by khosro10k** [reply]
I am working on a mobile application in the field of image processing and I am trying to find a faster way. I hope you can guide me
